### DISCUSSION TITLE
How to change the title of page dynamically in wintercms blog page

### DISCUSSION THREAD

**Discussion by Artful-dodger** [wintercms/winter#945] [Q&A]
I have been trying for a while to work out how to dynamiclly to append to blog page title for instance the first page title reads Drift-In Dot's corner  the second dynamically generated page should read as Drift-In Dot's corner-2 the same as the url, nothing I have tried has alter either the blog title name or the  blog/category title page if there is more than one dynamiclly generated page

**Comment by LukeTowers** [comment]
@Artful-dodger I'm unclear what you're trying to do exactly, are you trying to change the title that gets stored in the database or are you trying to change the `<title>` element in your theme?

**Reply by Artful-dodger** [reply]
hi Luke well the title is correct bu for SEO youshould not have dupicate titles so the the dynamiclly generated url is dot's-corner for the first blog page and for the second page it would be dot's-corner/2 but for the title its dot's-corner for first page and remains unchanged for the second page I am useing the following code to generate this result <title>{{ this.theme.website_name }} {{ this.page.meta_title|default(this.page.title) }}</title> and tried this {{ 'blog-post' | page({slug: 'slug-of-the-post'}) }} but nothing appends the vaule of the second page tile so I have a number of blog pages with the same title , i am looking for a method to increment the title inline with the URL
